### DISCUSSION TITLE
Remove

### DISCUSSION THREAD

**Discussion by Tptuaazn** [Zrips/CMI#4853] [Q&A]
How to remove < > in displayname.
![image](https://user-images.githubusercontent.com/74866833/103475871-99e88700-4de3-11eb-9187-2d4df5e81c43.png)

**Comment by variananora** [comment]
If you are using CMI Chat go to config.yml then 
```
Chat:
  ModifyChatFormat: true
  GeneralFormat: '{prefix}&f{displayName}&7: &r{message}'
  GroupFormat:
    '1': '{prefix}&f{displayName}&f: &r{message}'
    '2': '{prefix}&f{displayName}&7: &r{message}'
    '3': '{prefix}&f{displayName}&8: &r{message}'
```

edit that according to your formatting.
